oop/src/views/MoveInfoPage.vue

```diff
@@ -571,6 +579,10 @@ export default {
       this.postalCode = replaceSpecialCharacters(address.postalCode);
     },
     setFieldsToNull() {
+      // Set value of first address line to null
+      this.addressLines[0] = {
+        value: null,
+      }
```

Comment:
Do you also need the `id` property? Example here:

https://github.com/bcgov/MOH-AOP-OOP/blob/23405ff0afc3ae7ca144006d2c1d1af2faa18694/oop/src/views/MoveInfoPage.vue#L387-L391

Reply:
Attribute should be `id`, not `idVal`.